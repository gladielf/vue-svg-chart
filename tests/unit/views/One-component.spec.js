import { mount } from '@vue/test-utils';
import OneComponent from '@/views/One-component';

describe('One-component - Snapshot', () => {
  it('Snapshot', () => {
    const wrapper = mount(OneComponent);
    expect(wrapper.element).toMatchSnapshot();
  });
});
