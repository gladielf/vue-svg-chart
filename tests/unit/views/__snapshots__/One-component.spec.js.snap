// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`One-component - Snapshot Snapshot 1`] = `
<div
  class="home"
>
  <svg
    class="chart"
    height="8em"
    width="100%"
  >
    <g
      class="chart__group"
      stroke-linecap="round"
      stroke-width="0.5em"
    >
      <line
        class="chart__line"
        stroke="#ff5683"
        x1="6.25%"
        x2="6.25%"
        y1="95%"
        y2="5%"
      />
      <line
        class="chart__line chart__line--mod"
        stroke="#531af0"
        x1="6.25%"
        x2="6.25%"
        y1="95%"
        y2="5%"
      />
    </g>
    <g
      class="chart__group"
      stroke-linecap="round"
      stroke-width="0.5em"
    >
      <line
        class="chart__line"
        stroke="#ff5683"
        x1="18.75%"
        x2="18.75%"
        y1="95%"
        y2="5%"
      />
      <line
        class="chart__line chart__line--mod"
        stroke="#531af0"
        x1="18.75%"
        x2="18.75%"
        y1="95%"
        y2="5%"
      />
    </g>
    <g
      class="chart__group"
      stroke-linecap="round"
      stroke-width="0.5em"
    >
      <line
        class="chart__line"
        stroke="#ff5683"
        x1="31.25%"
        x2="31.25%"
        y1="95%"
        y2="5%"
      />
      <line
        class="chart__line chart__line--mod"
        stroke="#531af0"
        x1="31.25%"
        x2="31.25%"
        y1="95%"
        y2="5%"
      />
    </g>
    <g
      class="chart__group"
      stroke-linecap="round"
      stroke-width="0.5em"
    >
      <line
        class="chart__line"
        stroke="#ff5683"
        x1="43.75%"
        x2="43.75%"
        y1="95%"
        y2="5%"
      />
      <line
        class="chart__line chart__line--mod"
        stroke="#531af0"
        x1="43.75%"
        x2="43.75%"
        y1="95%"
        y2="5%"
      />
    </g>
    <g
      class="chart__group"
      stroke-linecap="round"
      stroke-width="0.5em"
    >
      <line
        class="chart__line"
        stroke="#ff5683"
        x1="56.25%"
        x2="56.25%"
        y1="95%"
        y2="5%"
      />
      <line
        class="chart__line chart__line--mod"
        stroke="#531af0"
        x1="56.25%"
        x2="56.25%"
        y1="95%"
        y2="5%"
      />
    </g>
    <g
      class="chart__group"
      stroke-linecap="round"
      stroke-width="0.5em"
    >
      <line
        class="chart__line"
        stroke="#ff5683"
        x1="68.75%"
        x2="68.75%"
        y1="95%"
        y2="5%"
      />
      <line
        class="chart__line chart__line--mod"
        stroke="#531af0"
        x1="68.75%"
        x2="68.75%"
        y1="95%"
        y2="5%"
      />
    </g>
    <g
      class="chart__group"
      stroke-linecap="round"
      stroke-width="0.5em"
    >
      <line
        class="chart__line"
        stroke="#ff5683"
        x1="81.25%"
        x2="81.25%"
        y1="95%"
        y2="5%"
      />
      <line
        class="chart__line chart__line--mod"
        stroke="#531af0"
        x1="81.25%"
        x2="81.25%"
        y1="95%"
        y2="5%"
      />
    </g>
    <g
      class="chart__group"
      stroke-linecap="round"
      stroke-width="0.5em"
    >
      <line
        class="chart__line"
        stroke="#ff5683"
        x1="93.75%"
        x2="93.75%"
        y1="95%"
        y2="5%"
      />
      <line
        class="chart__line chart__line--mod"
        stroke="#531af0"
        x1="93.75%"
        x2="93.75%"
        y1="95%"
        y2="5%"
      />
    </g>
  </svg>
</div>
`;
